mountinfo/mountinfo_unix.go

```diff
@@ -0,0 +1,35 @@
+// +build linux freebsd,cgo
+
+package mountinfo
+
+import (
+	"path/filepath"
+
+	"golang.org/x/sys/unix"
+)
+
+func mounted(path string) (bool, error) {
+	var st unix.Stat_t
+
+	err := unix.Lstat(path, &st)
+	switch err {
+	case unix.ENOENT:
+		// Nonexistent path, so not a mount point.
+		return false, nil
```

Comment:
I had some concern that a "broken" FUSE mount might hit this case, but it returns `ENOTCONN`. :+1:

(Just including a note in case anyone else reading this has the same question/concern. :sweat_smile:)